lib/templates/text.mustache

```diff
@@ -1,4 +1,4 @@
-<div class="res-text usertext-body">
+<div class="res-text usertext-body res-expando-host-{{moduleId}}">
```

Comment:
I think this better be placed in the expando-box element, perhaps by setting attribute `host`.